spring-boot/admin/src/main/resources/messages.properties

```diff
@@ -178,7 +178,9 @@ admin.help.getting.started.service.url.text= Please insert the URL to which the
 admin.help.getting.started.service.url.image.path=/images/coatrack-getting-started-service-url.png
 admin.help.getting.started.service.url.image.alt=CoatRack overview and first steps - Service URL
 admin.help.getting.started.service.identifier.heading=Unique identifier for your service
-admin.help.getting.started.service.identifier.text= Service users need to use this identifier as part of the URL so that the CoatRack gateway will know to which service this call should be forwarded.
+admin.help.getting.started.service.identifier.warning.heading=Service Identifier editing warning
+admin.help.getting.started.service.identifier.explanatory.text= Service users need to use this identifier as part of the URL so that the CoatRack gateway will know to which service this call should be forwarded.
+admin.help.getting.started.service.identifier.caution.text=Be careful about editing the service identifier! It will change the URL that is used to access the service via the CoatRack gateway which will make existing API keys unusable.
```

Comment:
The following part of the text is not correct, please remove it:
`[...] which will make existing API keys unusable.`

As there is a `.warning.heading`, maybe use `.warning.text` as key for this text?